ui/lib/core/addon/components/replication-secondary-card.js

```diff
@@ -10,22 +10,43 @@ import layout from '../templates/components/replication-secondary-card';
 
 export default Component.extend({
   layout,
-  state: computed('data', function() {
+  data: null,
+  dr: computed('data', function() {
     let dr = this.data.dr;
-    return dr && dr.state ? dr.state : 'unknown';
+    if (!dr) {
+      return false;
+    }
+    return dr;
+  }),
+  state: computed('data', function() {
+    return this.dr && this.dr.state ? this.dr.state : 'unknown';
   }),
   connection: computed('data', function() {
     return this.data.drStateDisplay ? this.data.drStateDisplay : 'unknown';
   }),
   lastWAL: computed('data', function() {
```

Comment:
if this property (and the other computed properties) are depending on `this.dr`, then this should be `computed('dr', function(){...})`

Reply:
Ah, good call.